pennylane/tape/tape.py

```diff
@@ -136,14 +136,18 @@ def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False):
     RY(0.2, wires=['a'])]
     """
     if depth == 0:
+        if iterative_call:
+            return tape._prep, tape._ops, tape._measurements
```

Comment:
The return signature for `iterative_call=True` is `QuantumTape -> (_prep, _ops, _measurements)`